README.md

```diff
@@ -20,6 +20,10 @@ The migration prototype currently handles exactly two cases:
 
 ### Setting up to run the prototype demonstration
 
+Install all the requirements of this demo via:
+
+    $ sudo yum-buildep ./demo/prototype-deps.spec
```

Comment:
Should be:

    sudo dnf builddep ./demo/prototype-deps.spec

Because pipsi isn't even in epel so this can't even work on CentOS/RHEL right now, but that we can fix with some switches